templates/credentials/your-badges.html

```diff
@@ -1,68 +1,100 @@
 {% extends "credentials/base_cred.html" %}
 
-{% block title %}Canonical Credentials -- Your Badges{% endblock %}
+{% block title %}Your Badges | Credentials{% endblock %}
```

Comment:
I get 500 error when going to this page (via direct URL, not sure where to find it in UI): https://ubuntu-com-13950.demos.haus/credentials/your-badges